### DISCUSSION TITLE
Can the iteration feature in the workflow exceed the limit of 30?

### DISCUSSION THREAD

**Discussion by Moiyu1998** [langgenius/dify#10436] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Can the iteration feature in the workflow exceed the limit of 30?

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by crazywoola** [comment]
https://github.com/langgenius/dify/blob/444c1f170a1a005256e12bac95cc993ae95bd9ea/api/configs/feature/__init__.py#L406-L429

Set those environment variables should able to get more steps.

I am not sure about the error you encountered, please provide more information, maybe a screenshot?

**Reply by Moiyu1998** [reply]
![image](https://github.com/user-attachments/assets/0886b3ad-7487-4224-bce0-49d1c168964d)

Sorry, I might have been mistaken. This might not be the iteration node limit, but rather the output length limit of the code execution.

**Reply by crazywoola** [reply, marked as answer]
https://github.com/langgenius/dify/issues/10439#issuecomment-2463616502

**Reply by Moiyu1998** [reply]
Thank you, this is very helpful to me.
